gpflow/features.py

```diff
@@ -132,6 +144,35 @@ def Kuu(feat, kern, *, jitter=0.0):
     return Kzz
 
 
+# ---------------------
+# InducingPatch feature
+# ---------------------
+
+
+class InducingPatch(InducingPointsBase):
+    pass
+
+
+@dispatch(InducingPatch, kernels.Convolutional, object)
+def Kuf(feat, kern, Xnew):
+    with params_as_tensors_for(feat, kern):
+        Xp = kern.get_patches(Xnew)  # N x num_patches x patch_len
+        bigKzx = kern.basekern.K(feat.Z, Xp)  # [M, N, P] -- thanks to broadcasting of kernels
+        Kzx = tf.reduce_sum(bigKzx * kern.weights if hasattr(kern, 'weights') else bigKzx, [2])
```

Comment:
1. Instead of `hasattr` you could create weighted convolutional kernel or set weights by default to None. This solution is too pythonic and straightforward.
2. It would be cool to have an output dimension here as well.